package.json

```diff
@@ -70,10 +71,11 @@
     "dev-integration": "nodemon --ext js,sol -x 'hardhat test test/integration/index.js --show-stack-traces'",
     "dev-unit": "nodemon --ext js,sol -x 'hardhat test test/unit/index.js --show-stack-traces'",
     "dev-ui": "hardhat run scripts/local-deploy.js",
-    "coverage": "NODE_OPTIONS='--max-old-space-size=8192' hardhat coverage",
-    "test": "hardhat test test/index.js",
-    "test-integration": "hardhat test test/integration/index.js",
-    "test-unit": "hardhat test test/unit/index.js",
+    "coverage": "NODE_OPTIONS='--max-old-space-size=8192' hardhat coverage --testfiles 'test/{unit,integration}/**/*.js'",
+    "test": "hardhat test test/unit/**/*.js test/integration/**/*.js",
+    "test-integration": "hardhat test test/integration/**/*.js",
+    "test-unit": "hardhat test test/unit/**/*.js",
+    "test-layout": "hardhat test test/layout/*.js",
```

Comment:
We left out the storage layouts from the default tests. We need to include them.